idaes/core/tests/test_control_volume_0d.py

```diff
@@ -729,6 +731,53 @@ def test_add_phase_component_balances_eq_rxns_no_idx():
         m.fs.cv.add_phase_component_balances(has_equilibrium_reactions=True)
 
 
+@pytest.mark.unit
+def test_add_phase_component_balances_in_rxns():
+    m = ConcreteModel()
+    m.fs = Flowsheet(default={"dynamic": False})
+    m.fs.pp = PhysicalParameterTestBlock()
+
+    # Set property package to contain inherent reactions
+    m.fs.pp._has_inherent_reactions = True
```

Comment:
In practice, this `_has_inherent_reactions` attribute would be set within the property package, correct?

In other words, the user wouldn't necessarily set this at the flowsheet level, right?

Reply:
Oh I thought I got rid of this comment. I saw you set the attribute in the build method of the generic property package...

Reply:
To answer anyway: Yes, this set by the property package. This was just for generic testing purposes.